.yarnclean

```diff
@@ -0,0 +1,57 @@
+# test directories
```

Comment:
I don't think it's really safe to remove these files... It could lead to side effects and bugs